packages/chart/src/CdcChart.jsx

```diff
@@ -747,6 +731,11 @@ export default function CdcChart({ configUrl, config: configObj, isEditor = fals
   let body = <Loading />
 
   if (!loading) {
+    const tableLink = (
```

Comment:
fix: align with additional media links. You might want to use the cove media controls hook here.

Reply:
I did not move that link at all and the ticket did not say to do any alignment changes.  It was already like that.  If you want that changed, how do you want it aligned specifically?